tests/by-util/test_ls.rs

```diff
@@ -558,6 +558,108 @@ fn test_ls_long_ctime() {
     }
 }
 
+#[test]
+#[cfg(not(windows))]
+// This test is currently failing on windows
+fn test_ls_order_birthtime() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.make_file("test-birthtime-1").sync_all().unwrap();
```

Comment:
I think a small comment explaining why `sync_all` is there might be helpful.

Reply:
Will do